### DISCUSSION TITLE
Final Exam and Certification Badge

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#62] [Announcements]
Hi all,

I am so glad about all the nice feedback about my Deep Learning course. It's so nice to know that it was useful to tens of thousands of people!

The number one request was a final exam and course completion badge for LinkedIn!

Happy to share that that's now added!  So, if you'd like to test your knowledge with 25 questions, you can take the final exam here: 

https://lightning.ai/pages/ai-education/deep-learning-fundamentals/certification/

Have fun!!

(It's also accessible from the bottom menu as shown below.)
<img width="235" alt="Screenshot 2023-06-16 at 9 21 40 AM" src="https://github.com/Lightning-AI/dl-fundamentals/assets/5618407/a9bc633b-4a6c-49cf-84d7-f9123e373190">

**Comment by rsousa95** [comment]
Hi! I already did the exam some time ago and was trying to access it and the certificate again but I'm not being able to. Clicking the "Final certification exam" button just refreshes the page, and the link you have sends me to an empty page. Any idea on how to access it?

**Reply by eulerianKnight** [reply]
Hey, I am facing the same issue. Any update on this?

**Reply by rasbt** [reply]
Thanks for the note, looking into this!

**Reply by eulerianKnight** [reply]
Thank you Sebastian for the help, it's resolved now.

**Reply by rasbt** [reply]
Awesome, glad to hear!

**Comment by gnsepili** [comment]
Hello! I took the exam and finished it. However, after completing the quiz successfully, it prompts me to share the certificate on LinkedIn/Twitter. When I click on LinkedIn, it redirects me to add the certificate on the LinkedIn platform with Credential ID 1. But when I click on the certificate URL, it only redirects back to the course site, and I cannot access my certificate.
![Screenshot 2024-06-21 021924](https://github.com/Lightning-AI/dl-fundamentals/assets/32766800/bb0a12a9-e1f8-41ae-bcea-e1fbd0a5bcad)

**Reply by SomaRe** [reply]
Facing the same issue

**Reply by rasbt** [reply]
Sorry about this issue. Our web developer who developed the certificate feature is no longer at Lightning but I will see if I can find someone who can look into this.

**Reply by Sanju-Shrestha** [reply]
Hello Sebastian. I recently completed the course and the certification exam. I am facing a similar issue regarding the certification. This was the link generated [link](https://lightning.ai/courses/deep-learning-fundamentals/).

**Reply by dcjkfgdjhd** [reply]
> Sorry about this issue. Our web developer who developed the certificate feature is no longer at Lightning but I will see if I can find someone who can look into this.

Hello, the problem still exists unfortunately, just passed the exam with 96% score, but I also only get "certificate ID = 1" and a link that only links back to the course page, not a certificate. 

Would be great if it could be fixed, in the current state the certificate looks... suspicious.

By the way, the course itself is truly excellent, thank you very much!
